scylla/src/response/query_result.rs

```diff
@@ -444,6 +458,1180 @@ impl QueryRowsResult {
 
         (raw_rows_with_metadata, tracing_id, warnings)
     }
+
+    pub fn rows_displayer(&self) -> RowsDisplayer<'_> {
+        RowsDisplayer::new(self)
+    }
+}
+
+/// A utility struct for displaying rows received from the database in a [`QueryRowsResult`].
```

Comment:
:pushpin: As I noted elsewhere, we should consider making this work both with `QueryRowsResult` and with `TypedRowStream`.